sdk/eventgrid/eventgrid/CHANGELOG.md

```diff
@@ -1,21 +1,21 @@
 # Release History
 
-## 4.6.1 (Unreleased)
+## 4.7.0 (2022-02-08)
 
-### Features Added
-
-### Breaking Changes
+### Key Bug Fixes
 
-### Bugs Fixed
+- The TypeScript typings for two events have had small changes to accurately reflect the data sent by Azure.
+  - `Microsoft.EventHub.CaptureFileCreated`'s `fileurl` property is now correctly cased as `fileUrl`
+  - `Microsoft.Storage.DirectoryDeleted`'s `recursive` property has been changed from `boolean` to `string`. The service sets this property to the string `"true"` or `"false"`.
 
-### Other Changes
+In both cases, the typescript typings now correctly reflect the shape of the event.
```

Comment:
nit

```suggestion
In both cases, the TypeScript typings now correctly reflect the shape of the event.
```

Reply:
though honestly I'm not sure if this sentence is needed

Reply:
You're right.  I removed it.